### ISSUE TITLE
Incorrect SpaceBeforeBrace lint error

### ISSUE THREAD

**Issue by ivancuric** [brigade/scss-lint#682]
The linter often gets confused, and the only solution is to delete the brace and space, let the linter parse, and then retype the brace again. I don't see any pattern on when the bug occurs.

```Bash
main.scss:4727 [W] SpaceBeforeBrace: Opening curly brace `{` should be preceded by one space
```
![image](https://cloud.githubusercontent.com/assets/2827047/12451160/037d896c-bf88-11e5-91b8-0565fdd2064f.png)

**Comment by bartveneman** [comment]
Are you sure this is an Scss-lint issue and not one for the editor plugin you're using? Does the error also occur when you run scss-lint from the command line?

**Comment by ivancuric** [comment]
Yep, the error was from the console output.

**Comment by sds** [comment]
Hey @ivancuric,

Running the following code against `scss-lint` with the default configuration:

```scss
.installment-accordion-row {
  display: none;
  &.is-visible {
    display: table-row;
  }
}

.installment-accordion-toggle {
  color: $b1;
}
```

...I get the following correct output:

```
test.scss:3 [W] EmptyLineBetweenBlocks: Rule declaration should be preceded by an empty line
```

Thus nothing seems wrong here. The "confusion" you're referring to must be a quirk of the editor you're using (or the scss-lint plugin for that editor, neither of which this project maintains).

**Comment by ivancuric** [comment]
The console output was from the `scss-lint` command, not the editor. I'll see if the bug reappears and get back with some more helpful info. I've no idea what caused it.

**Comment by sds** [comment]
Cool, let us know if new information presents itself.

**Comment by ivancuric** [comment]
![screen shot 2016-01-22 at 00 04 41](https://cloud.githubusercontent.com/assets/2827047/12497455/34c84f24-c09c-11e5-9298-168bb1929f77.png)
In certain cases, Emmet inserts a non-breaking space instead of a regular space :)

**Comment by sds** [comment]
Thanks for reporting your findings, @ivancuric. It will help others if they run into this issue as well.

**Comment by ivancuric** [comment]
It was a problem with using `option+space` when typing fast on on osx. Disabled it using Karabiner:
![image](https://cloud.githubusercontent.com/assets/2827047/12612231/73d7984c-c4f1-11e5-9edd-0772374830db.png)
